### PR TITLE
Added hint that original file has to exist

### PR DESCRIPTION
**Pull request by pogosheep**
Thanks to Mathias Brodala for the hint, the issue can be found here:
- https://forge.typo3.org/issues/56148
- https://forge.typo3.org/issues/47996

### CODE REVIEW

File: Documentation/Internationalization/Translation/Index.rst [hunk 1]
```diff
@@ -101,6 +101,9 @@ and the result can be easily seen in the backend:
 
    - The files containing the custom labels must be located inside an extension. Other locations
      will not be considered (this is a bug, but must be taken as a constraint for now).
+     
+   - The original translation needs to exist in :code:`typo3temp/l10n/` for this to work
```
Comment (mbrodala) [inline Documentation/Internationalization/Translation/Index.rst]: Either this or next to the base translation file in extensions, e.g. `typo3conf/ext/myext/Resources/Private/Language/`

**Comment by marble** [comment]
Hello Mathias (mbrodala), I don't understand: What do you mean? Do you agree with Daniel?

**Comment by mbrodala** [comment]
@marble Yes, this hint is useful until the actual bug is fixed. My comment only added that the original translation can live in two locations for overrides to take effect.

**Comment by marble** [comment]
Thanks! I've rephrased it a bit so It's even clearer I hope. Daniel, the ReST code in our official manuals is very complicated indeed. But ReST is very simple if you use it for the "normal" things like text, lists, links and so on yourself.

**Comment by marble** [comment]
Ah, here's the direct link: http://docs.typo3.org/typo3cms/CoreApiReference/Internationalization/Translation/Index.html#xliff-translating-custom

**Comment by pogosheep** [comment]
Thanks Martin. Thanks for all the commitment and fast but quality work.

**Pull request merged**